### ISSUE TITLE
Install guide?

### ISSUE THREAD

**Issue by FormalSnake** [Coopydood/HyperFluent-GRUB-Theme#1]
Hi, I come from macOS and I want to use EndeavourOS (Arch) on my PC, but the official Arch guide left me confused on what to do.
Some sort of resource or tutorial in the readme would be very nice.

I was following this [guide](https://wiki.archlinux.org/title/GRUB/Tips_and_tricks)

**Comment by Miguel-hrvs** [comment]
https://wiki.archlinux.org/title/GRUB/Tips_and_tricks

3.3 section

**Comment by FormalSnake** [comment]
Yes, I managed to figure it out but it would be nice to have instructions

**Comment by Coopydood** [comment]
I'll add a quick install guide on next update!

**Comment by FormalSnake** [comment]
Something I noticed, when using Displayport on a widescreen momitor the GRUB breaks, but when using HDMI it scales correctly but is very laggy.

**Comment by raffaem** [comment]
> Once done you have to write in the terminal sudo update-grub and reboot.

I believe it's `sudo grub-mkconfig -o /boot/grub/grub.cfg`

**Comment by fmaximus** [comment]
Small question,  for people using root fs encryption, wouldn't it be necessary to put the files in the unencrypted /boot partition.
